src/cryptonote_core/cryptonote_tx_utils.h

```diff
@@ -101,6 +114,10 @@ namespace boost
       a & x.amount;
       a & x.rct;
       a & x.mask;
+      a & x.multisig_kLRki;
+      if (ver < 1)
+        return;
+      a & x.real_out_additional_tx_keys;
```

Comment:
Shouldn't `a & x.multisig_kLRki;` come after the `ver < 1` check?

BTW I suppose `x. real_out_additional_tx_keys` being missing previously was a bug which is fixed here.